### PR TITLE
L1T: updates for firmware testing and unpacker update

### PR DESCRIPTION
**Pull request by bundocka**
- Updates CaloL2 MP unpacker to use correct jet quality bit
- Various updates to the CaloL2 firmware testing code
- Update the script that converts the offline calorimeter LUTs to the online xml files

### CODE REVIEW

File: EventFilter/L1TRawToDigi/plugins/implementations_stage2/MPUnpacker_0x10010033.cc [hunk 1]
```diff
@@ -227,7 +227,7 @@ namespace l1t {
         jet.setHwEta(CaloTools::caloEta(mpEta));
         jet.setHwPhi((raw_data >> 6) & 0x7F);
         jet.setHwPt((raw_data >> 13) & 0xFFFF);
-        jet.setHwQual((raw_data >> 29) & 0x1);
+        jet.setHwQual((raw_data >> 30) & 0x1);
```
Comment (aloeliger) [inline EventFilter/L1TRawToDigi/plugins/implementations_stage2/MPUnpacker_0x10010033.cc:230]: out of curiosity, does this change mean that the current unpacker is reading the wrong quality bit?

Comment (bundocka) [reply EventFilter/L1TRawToDigi/plugins/implementations_stage2/MPUnpacker_0x10010033.cc:230]: yes, but this is the MP unpacker, so it isn't used anywhere (except for local firmware testing). the demux unpacker correctly unpacks the jet quality already, and is used for DQM, L1 Ntuples etc.

**Comment by aloeliger** [comment]
+l1

**Comment by perrotta** [comment]
+1

**Pull request merged**